src/app/contexts/RequestContext/index.jsx

```diff
@@ -31,18 +46,15 @@ export const RequestContextProvider = ({
 };
 
 RequestContextProvider.propTypes = {
+  bbcOrigin: string,
   children: node.isRequired,
-  env: string,
   id: string,
+  isAmp: bool.isRequired,
```

Comment:
Just thinking, it would be nice if `isAmp` was optional and had a default of `false`.
```jsx
// we would have
<RequestContextProvider />
// instead of
<RequestContextProvier isAmp={false} />
```
If `isAmp` was made required so that we can be explicit about it while using `RequestContextProvider` you can safely ignore my comment.

Reply:
Yeah isAmp is passed from high up in the app, so we should keep it required ideally, even though it does make it a bit more annoying in tests